src/components/DataTable/DataTableTotalResults.vue

```diff
@@ -32,7 +32,11 @@ const totalRows = computed(() => {
   if (typeof props.total !== 'number') {
     return '-';
   }
-  return props.total.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  const commaFormatted = props.total
+    .toString()
+    .replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  // When our ES total is 10,000, we want to display 10,000+.
+  return props.total === 10000 ? `${commaFormatted}+` : commaFormatted;
```

Comment:
if there are say 15054... wouldn't this just show `15,054+`?

Reply:
oh or does props have a max of 10000? maybe just in case:
`return props.total >= 10000 ? "10,000+" : commaFormatted;`